adding commands directly to docs (#1022)
temporalio/documentation: docs/clusters/how-to-integrate-elasticsearch-into-a-temporal-cluster.md

Run the following commands to create the index schema and index:

```bash

> # ES_SERVER is the URL of Elasticsearch server i.e. "http://localhost:9200".

Reviewer: ```suggestion
# ES_SERVER is the URL of Elasticsearch server; for example, "http://localhost:9200".
```
